cloud_topic: /fastlio2/body_cloud
odom_topic: /fastlio2/lio_odom
map_frame: map
local_frame: lidar

key_pose_delta_deg: 10
key_pose_delta_trans: 0.5
loop_search_radius: 1.0
loop_time_tresh: 60.0
loop_score_tresh: 0.15
loop_submap_half_range: 5
submap_resolution: 0.1
min_loop_detect_duration: 5.0

grid_2d_z_min: -0.2
grid_2d_z_max: 1.0
grid_2d_resolution: 0.05